File: app/src/main/java/org/kd/main/common/entities/Bank.java
package org.kd.main.common.entities;

import lombok.*;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
public class Bank implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String name;
    private String shortname;

    public Bank(String name, String shortname) {
        this.name = name;
        this.shortname = shortname;
    }

    @Override
    public String toString() {
        return name;
    }

}

File: C#UI/Banque/Common/Entities/Bank.cs
using System;

namespace Banque.Common.Entities
{
    public class Bank
    {
        public long Id { get; set; }

        public String Name { get; set; }
        public String Shortname { get; set; }

        public Bank()
        {
        }

        public Bank(long id, String name, String shortname)
        {
            this.Id = id;
            this.Name = name;
            this.Shortname = shortname;
        }

        public Bank(String name, String shortname)
        {
            this.Name = name;
            this.Shortname = shortname;
        }

        override
        public String ToString()
        {
            return Name;
        }
    }
}
